aws-lc-rs/src/signature.rs

```diff
@@ -390,133 +397,133 @@ impl<B: AsRef<[u8]>> UnparsedPublicKey<B> {
 }
 
 /// Verification of signatures using RSA keys of 1024-8192 bits, PKCS#1.5 padding, and SHA-1.
-pub static RSA_PKCS1_1024_8192_SHA1_FOR_LEGACY_USE_ONLY: RsaParameters = RsaParameters(
+pub static RSA_PKCS1_1024_8192_SHA1_FOR_LEGACY_USE_ONLY: RsaParameters = RsaParameters::new(
```

Comment:
NP: Should we migrate these `pub static` variables to instead be `pub const`?

Reply:
I'm going to do this one is a separate PR, mainly as moving these to `const` will require moving the `digest` module statics to const as well.